### ISSUE TITLE
FNX3-15659 ⁃ [Bug] UnknownFormatConversionException when opening sync settings and language is not English

### ISSUE THREAD

**Issue by cadeyrn** [mozilla-mobile/fenix#1580]
### Steps to reproduce
1. choose German as language in your Android settings
1. make sure you're signed in 
1. open sync settings

### Expected behavior
You see your sync settings.

### Actual behavior
Fenix crashes.

```
2019-04-11 23:17:53.140 6602-6602/org.mozilla.fenix.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.debug, PID: 6602
    java.util.UnknownFormatConversionException: Conversion = '@'
        at java.util.Formatter$FormatSpecifier.conversion(Formatter.java:2781)
        at java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2811)
        at java.util.Formatter$FormatSpecifierParser.<init>(Formatter.java:2624)
        at java.util.Formatter.parse(Formatter.java:2557)
        at java.util.Formatter.format(Formatter.java:2504)
        at java.util.Formatter.format(Formatter.java:2458)
        at java.lang.String.format(String.java:2814)
        at android.content.res.Resources.getString(Resources.java:488)
        at androidx.fragment.app.Fragment.getString(Fragment.java:854)
        at org.mozilla.fenix.settings.AccountSettingsFragment.updateLastSyncedTimePref(AccountSettingsFragment.kt:135)
        at org.mozilla.fenix.settings.AccountSettingsFragment.updateLastSyncedTimePref$default(AccountSettingsFragment.kt:124)
        at org.mozilla.fenix.settings.AccountSettingsFragment.onCreatePreferences(AccountSettingsFragment.kt:58)
        at androidx.preference.PreferenceFragmentCompat.onCreate(PreferenceFragmentCompat.java:160)
        at org.mozilla.fenix.settings.AccountSettingsFragment.onCreate(AccountSettingsFragment.kt:32)
        at androidx.fragment.app.Fragment.performCreate(Fragment.java:2528)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:843)
        at androidx.fragment.app.FragmentManagerImpl.addAddedFragments(FragmentManagerImpl.java:2092)
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1866)
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1822)
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1723)
        at androidx.fragment.app.FragmentManagerImpl$1.run(FragmentManagerImpl.java:146)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:180)
        at android.app.ActivityThread.main(ActivityThread.java:6950)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:835)
2019-04-11 23:18:41.156 6602-6647/org.mozilla.fenix.debug E/GeckoConsole: [JavaScript Error: "1555017521155	Telemetry::CoveragePing	ERROR	no endpoint base set" {file: "resource://gre/modules/Log.jsm" line: 679}]
    append@resource://gre/modules/Log.jsm:679:12
    log@resource://gre/modules/Log.jsm:360:16
    error@resource://gre/modules/Log.jsm:368:10
    startup@resource://gre/modules/CoveragePing.jsm:40:11
    setupTelemetry/this._delayedInitTask<@resource://gre/modules/TelemetryController.jsm:721:30
```

### Device information
* Android device: HTC U11 / Android 8.0
* Fenix version: Fenix revision 9de120b8ab793794190617df93cc751470497121

**Comment by cadeyrn** [comment]
FWIW, it no longer crashes for me, but since it's still the same build it would still be interesting to know what caused the crash.

**Comment by cadeyrn** [comment]
The crash only happens with German as language in your Android settings, but not with English (I didn't test other languages). I updated the STR.

**Comment by kbrosnan** [comment]
Sentry issue: [FENIX-FT](https://sentry.prod.mozaws.net/operations/fenix/issues/5532280/)

**Comment by cadeyrn** [comment]
The cause is an error in the German translation:

`<string name="sync_last_synced_summary">Letzte Synchronisation: %@</string>`

should be

<string name="sync_last_synced_summary">Letzte Synchronisation: %s</string>

I verified in my local build that this crash fixes the crash and suggested a change via Pontoon.

**Comment by cadeyrn** [comment]
Fixed by #1763.
